py-pkgs/08-ci-cd.ipynb

```diff
@@ -820,7 +811,22 @@
     "alt: Adding the PyPI API token to our GitHub repository.\n",
     "---\n",
     "Adding the PyPI API token to our GitHub repository.\n",
-    "```"
+    "```\n",
+    "\n",
+    "Finally, the last step in our CD workflow will be to create a GitHub release that corresponds to the new version published on TestPyPI and PyPI. \n",
+    "To do this we will a second GitHub action from PSR, [python-semantic-release/upload-to-gh-release@main](https://github.com/python-semantic-release/upload-to-gh-release). \n",
```

Comment:
Missing word "use" : "To do this we will _use_ a second GitHub action from PSR..."